xml/art_sle_modules_quick.xml

```diff
@@ -89,9 +87,10 @@
     </listitem>
    </varlistentry>
   </variablelist>
+ </sect1>
 
-  <sect2 xml:id="sec-modules-what-modules">
-   <title>Modules Available for the &sle; Product Family</title>
+ <sect1 xml:id="sec-modules-what-modules">
```

Comment:
I would shorten the titles to 'Extensions for the [...]' and 'Modules for the [...]'.